tgui/packages/tgui/interfaces/NTRecruiter.js

```diff
@@ -80,8 +80,9 @@ export const NTRecruiter = (props, context) => {
           >
             <LabeledList>
               <LabeledListItem label="1#" color="silver">
-                To win this game you must hire/dismiss <b>{total_curriculums}</b> candidates,
-                one wrongly made choice leads to a game over.
+                To win this game you must hire/dismiss{' '}
```

Comment:
You've got some spurious changes in this PR, see if you can't undo these

Reply:
These are fine, they will go away as diffs once my PR is merged